artorious/WeConnect
# Add static UI to WeConnect - a web application

#### What does this PR do?

Add a static user interface to a web application

#### Description of Task to be completed?

Simulate the functionality of a website platform creates awareness for businesses.
Give users the ability to write reviews about the businesses they have interacted with. 
 
#### How should this be manually tested?

Visit the website: https://artorious.github.io/
Download or Clone: https://github.com/artorious/WeConnect.git

Navigate the website to ensure logical flow through the pages and links View the website on different sized screens to ensure responsiveness

A user should be able to log in, register a business account/ profile or prompted to sign up first if not registered. A signed in user should be able to Update a business profile

#### What are the relevant pivotal tracker stories?

[#155504557](https://www.pivotaltracker.com/story/show/155504557)
[#155504485](https://www.pivotaltracker.com/story/show/155504485)
[#155504574](https://www.pivotaltracker.com/story/show/155504574)

#### Any background context you want to add?

Target or expected users include business owners and their customers

#### Screenshots
![screenshot from 2018-02-28 19 26 21](https://user-images.githubusercontent.com/15730750/36799579-8d53ea2e-1cbe-11e8-94b7-c494c207a122.png)

![screenshot from 2018-02-28 19 27 30](https://user-images.githubusercontent.com/15730750/36799590-93f77f8a-1cbe-11e8-92e4-8395bb4d120f.png)

![screenshot from 2018-02-28 19 30 08](https://user-images.githubusercontent.com/15730750/36799595-94f8a6a2-1cbe-11e8-8a93-e5263cfb1184.png)

![screenshot from 2018-02-28 19 29 13](https://user-images.githubusercontent.com/15730750/36799594-94b9f7c2-1cbe-11e8-807d-b41c692277be.png)

![screenshot from 2018-02-28 19 28 51](https://user-images.githubusercontent.com/15730750/36799593-948090fe-1cbe-11e8-8b54-3a7800e21795.png)

![screenshot from 2018-02-28 19 28 09](https://user-images.githubusercontent.com/15730750/36799592-9449eebe-1cbe-11e8-8df7-f2300ac60e4c.png)
